readme.md

```diff
@@ -47,13 +39,13 @@ Lile comes with a 'generator' to quickly generate new Lile services.
 
 Lile follows Go's conventions around `$GOPATH` (see [How to Write Go](https://golang.org/doc/code.html#Workspaces)) and is smart enough to parse your new service's name to create the service in the right place.
 
-If your Github username was `lileio` and you wanted to create a new service for posting to Slack you might use the following command.
+If your Github username was `tessthedog` and you wanted to create a new service for posting to Slack you might use the following command.
 
 ```
-lile new lileio/slack
+lile new --name tessthedog/slack
 ```
 
-This will create a project in `$GOPATH/src/github.com/lileio/slack`
+Follow the command line instructions and this will create a new project folder for you with everything you need to continue.
```

Comment:
I changed this since to better reflect the fact that I think Lile outputed the file in your $GOPATH in older versions but doesn't anymore.

The CLI output clearly tells you what it did though so I think this is ok?